app/uk/gov/hmrc/apidefinition/controllers/controllers.scala

```diff
@@ -45,7 +45,7 @@ package object controllers {
       {
         result =>
           result.fold(
-            errors => successful(UnprocessableEntity(validationResult(errors))),
+            errors => successful(UnprocessableEntity(validationResult(errors.toSeq))),
```

Comment:
I don't know what the way forward should be for these interactions with Play JSON but I have done things like this in other repos

`successful(UnprocessableEntity(validationResult(errors.map(error => (error._1, error._2.toSeq)).toSeq)))`

Reply:
That's much nicer than the way that I tried to get rid of mutable `Seq`s. I've rolled your suggestions into this PR